guides/source/developers/getting-started/customer-flow.html.md

```diff
@@ -0,0 +1,247 @@
+# Customer flow
+
+This article outlines a typical customer flow in Solidus on a technical level.
+The [`Spree::Order`][spree-order] model ties much of this flow together.
+However, every order placed touches many Solidus models.
+
+Note that this article does not detail _every_ single interaction that a Rails
+server logs. Nor does it detail every single attribute on every single relevant
+model. For example, it does not mention the `special_instructions` that a
+customer might add to an order, or any of the order cancellation process.
+
+## Standard flow
+
+Customer arrives at the store's URL.
+
+- Solidus determines whether the customer is _new_ or _returning_.
+  - If returning, Solidus determines whether the customer is a guest.
+  - If customer was logged in during a previous session, Solidus determines
+    whether customer should be logged in again.
+    - If new, `guest_token` cookie might be generated for the customer.
```

Comment:
too much indent?

Reply:
Thank you. This was intentional, but I think that it makes more sense to indent these lists this way:

```
 - If returning, Solidus determines whether the customer is a guest.
    - If customer was logged in during a previous session, Solidus determines
      whether customer should be logged in again.
  - If new, `guest_token` cookie might be generated for the customer.
```

Because the "If customer was logged in..." item relates specifically to the returning customers item above.